cpp/oneapi/dal/algo/kmeans/BUILD

```diff
@@ -41,6 +52,7 @@ dal_test_suite(
 dal_test_suite(
     name = "tests",
     host_tests = [
+        ":common_tests",
         ":cpu_tests",
     ],
     dpc_tests = [
```

Comment:
Why you do not include common tests to the `dpc_tests` if they are really common?

Reply:
It seems to me, common tests should be able to run on host only (w/o GPU)

Reply:
yes, from my perspective in common tests we are testing common parts/branches for CPU/GPU (like descriptor) on host, i.e. we can avoid launch on GPU.

Reply:
We have two interfaces of oneDAL: host and dpc++. When you say "common tests," I think these tests shall work for both interfaces - okay, if you will test in only on CPUs for dpc++ interface. But seems like you are testing the host interface only.

Reply:
made tests renaming